index.js

```diff
@@ -4,6 +4,8 @@ import VizbeeVideo from './VizbeeVideo';
 import VizbeeVideoStatus, {VizbeePlayerState} from './VizbeeVideoStatus';
 import VizbeeTrackStatus from './VizbeeTrackStatus';
 import VizbeeTrackInfo from './VizbeeTrackInfo';
+import VizbeeSignInDelegate from './VizbeeSignInDelegate';
+import VizbeeSignInInfo from './VizbeeSignInInfo';
```

Comment:
Let's add these imports after VizbeeManager

Reply:
Re-organized the imports as per flow